app/codeception/acceptance/HomeCest.php

```diff
@@ -17,6 +17,53 @@ public function _after(\AcceptanceTester $I)
     public function openHomepage(\AcceptanceTester $I)
     {
         $I->amOnPage('/');
-        $I->see('Symlex');
+        $I->see('Welcome to Symlex');
     }
+
+    public function login(\AcceptanceTester $I)
+    {
+        $I->amOnPage('/');
```

Comment:
Should be /welcome (same for the other tests, depending on which page you want to be)